### PR TITLE
chromium: update to 83.0.4103.61

### LINKED ISSUE #377: chromium: update to 83.0.4103.61
Stable release announcements since our latest update:

- https://chromereleases.googleblog.com/2020/05/stable-channel-update-for-desktop_19.html

Targets and platforms we need to check:

**chromium-x11:**
- [ ] zeus, clang, MACHINE="intel-core2-32"
- [ ] zeus, clang, MACHINE="intel-corei7-64"
- [ ] warrior, clang, MACHINE="raspberrypi3"
- [ ] warrior, clang, MACHINE="intel-corei7-64"

**chromium-ozone-wayland:**

- [ ] zeus, clang, MACHINE="m3ulcb"
- [ ] zeus, clang, MACHINE="qemuarm64"
- [ ] warrior, clang, MACHINE="raspberrypi3"

### PR DESCRIPTION
**Pull request by msisov**
Stable release announcements since our latest update:
https://chromereleases.googleblog.com/2020/05/stable-channel-update-for-desktop_19.html

Test-built with:

License updates:

Updated licences:
* third_party/android_deps/libs/com_google_android_gms_play_services_*
  - GMSCore: Rolling Google Play Services 15.0.1 -> 17.0.0
  - https://crrev.com/c/2083221
* third_party/android_deps/libs/com_google_protobuf_protobuf_javalite/LICENSE
  - Roll protobuf javalite runtime + fix gn package name
  - https://crrev.com/c/1762684
* third_party/ffmpeg/CREDITS.chromium
  - M82 FFMpeg configuration updat
  - https://crrev.com/c/206563
* third_party/perfetto/NOTICE -> */LICENSE
  - Updated perfetto
  - https://crrev.com/c/2134619

Added licences:
* third_party/android_provider/LICENSE
  - Upstream internal DownloadCollectionBridge code
  - https://crrev.com/c/2090011
* third_party/pywebsocket3/src/LICENSE
  - Added pywebsocket3 to third_party
  - https://crrev.com/c/2059041
* third_party/schema_org/LICENSE
  - Add Schema.org schema file for eventual use in document metadata extractor
  - https://crrev.com/c/2023864
* third_party/weston/LICENSE
  - Add Weston to third_party.
  - https://crrev.com/c/1962128

Removed licenses:
* third_party/gestures/LICENSE && third_party/libevdev/LICENSE
  - Removed gestures and libevdev from third_party
  - https://crrev.com/c/2048906

* chromium-x11
  * zeus, MACHINE="intel-corei7-64"
  * zeus, MACHINE="intel-core2-32"
  * warrior, MACHINE="intel-corei7-64"
* chromium-ozone-wayland:
  * zeus, MACHINE="qemuarm64"
  * zeus, MACHINE="m3ulcb"
  * warrior, MACHINE="raspberrypi3"

Fixes #377
Signed-off-by: Maksim Sisov <msisov@igalia.com>

### CODE REVIEW

File: recipes-browser/chromium/chromium.inc [hunk 1]
```diff
@@ -33,6 +33,7 @@ inherit pythonnative
 # currently contains Chromium's license (BSD-3-Clause) as well as Blink's
 # (LGPL2.0, LGPL2.1 and BSD-2-Clause).
 LICENSE = "BSD-3-Clause & LGPL-2.0 & LGPL-2.1"
+
```
Comment (rakuco) [inline recipes-browser/chromium/chromium.inc:36]: This doesn't look necessary.

File: recipes-browser/chromium/chromium.inc [hunk 2]
```diff
@@ -18,7 +18,7 @@ TOOLCHAIN = "clang"
 # to build the native recipes (e.g. GN) with clang too.
 TOOLCHAIN_class-native = "clang"
 
-inherit pythonnative
+inherit python3native
```
Comment (rakuco) [inline recipes-browser/chromium/chromium.inc:21]: Can you explain this change?

File: recipes-browser/chromium/chromium.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium.inc:36]: Hmm. I don't know how it got here.

File: recipes-browser/chromium/chromium.inc (hunk 2 above)
Comment (msisov) [reply recipes-browser/chromium/chromium.inc:21]: Yeah, fixes " pkg_resources not found" (I don't remember the exact error message, but it was something like that.).

File: recipes-browser/chromium/chromium.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium.inc:36]: fixed

File: recipes-browser/chromium/chromium.inc (hunk 2 above)
Comment (rakuco) [reply recipes-browser/chromium/chromium.inc:21]: Could you file an issue about this so we can investigate whether it is really necessary or not?

File: recipes-browser/chromium/chromium.inc (hunk 1 above)
Comment (rakuco) [reply recipes-browser/chromium/chromium.inc:36]: Did you push the fix? I can still see the line here.

Comment (msisov) [reply recipes-browser/chromium/chromium.inc:36]: Yes, I did. 

$ git push -f me m83
Everything up-to-date

Comment (rakuco) [reply recipes-browser/chromium/chromium.inc:36]: that's weird, I can see the line in your m83 branch too: https://github.com/msisov/meta-browser/blob/9687df6ce742ea9601afce9cb058337b74cd8d23/recipes-browser/chromium/chromium.inc#L36

File: recipes-browser/chromium/chromium.inc (hunk 2 above)
Comment (msisov) [reply recipes-browser/chromium/chromium.inc:21]: https://github.com/OSSystems/meta-browser/issues/380

File: recipes-browser/chromium/chromium.inc (hunk 1 above)
Comment (msisov) [reply recipes-browser/chromium/chromium.inc]: Tried to revert and push again. Check now.

**Pull request merged**